### ISSUE TITLE
Registration should create PDS

### ISSUE THREAD

**Issue by newgrounds** [HumanDynamics/openPDS-RegistryServer#2]
It isn't going to be possible for us to create a PDS manually for each user.

Imagine having thousands of users register.  You can register as a viewer of someone else's data or as a user with your own PDS.

**Comment by brian717** [comment]
I was under the impression that your personal data stores were living on one machine. If this is the case, users are lazily created upon first access.

**Comment by newgrounds** [comment]
Okay, but this is if all the users share a single PDS right?

**Comment by brian717** [comment]
Yes, this is if all users are served from the same PDS instance. We haven't built  automation to deploy a machine with a new PDS instance running upon registration. I believe that would actually be dependent on the hosting solution used as well. Is this an issue?

**Comment by newgrounds** [comment]
No, I believe this setup will be sufficient once scopes are implemented.
